test/peer-store/proto-book.spec.js

```diff
@@ -96,7 +95,7 @@ describe('protoBook', () => {
       const supportedProtocols = ['protocol1', 'protocol2']
 
       let changeCounter = 0
-      ee.on('change:protocols', () => {
+      peerStore.on('change:protocols', () => {
```

Comment:
I think we should add the events to the API docs at https://github.com/libp2p/js-libp2p/blob/0.28.x/doc/API.md#events. We could split that into subsections (`libp2p`, `libp2p.peerStore`) to link to from the TOC. WDYT?

Reply:
Yes, that makes sense.  Good call